dogs.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="zxx">
+   <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <title>Adopt A Dog</title>
+      <link rel="stylesheet" href="stylesheet/style.css">
+   </head>
+   <body>
+      <header>
+         <div class="header-container">
+            <div class="logo-bar">
+               <img src="images/adoptadog-logo.svg" alt="">
+            </div>
+            <div class="navigation-bar">
+               <nav>
+                  <ul>
+                     <li><a href="index.html">Home / </a></li>
+                     <li> <a href="dogs.html">Dogs / </a></li>
+                     <li><a href="blog.html">Blog </a></li>
+                  </ul>
+               </nav>
+            </div>
+         </div>
+      </header>
+      <div >
+         <h2>Here are all of our Dogs!</h2>
+      </div>
+      <div class="dogs-container">
```

Comment:
great! This organization is a good direction. Think of what is a div and what is a section and maybe leverage sections more